data/LogicHelpers.json

```diff
@@ -111,13 +111,13 @@
         (lacs_condition == 'medallions' and has_medallions(lacs_medallions)) or
         (lacs_condition == 'dungeons' and has_dungeon_rewards(lacs_rewards)) or
         (lacs_condition == 'tokens' and (Gold_Skulltula_Token, lacs_tokens)))",
-    "can_receive_ganon_bosskey": "(
-        (shuffle_ganon_bosskey == 'stones' and has_stones(ganon_bosskey_stones)) or
+    "can_receive_ganon_bosskey": "(shuffle_ganon_bosskey == 'stones' and has_stones(ganon_bosskey_stones)) or
         (shuffle_ganon_bosskey == 'medallions' and has_medallions(ganon_bosskey_medallions)) or
         (shuffle_ganon_bosskey == 'dungeons' and has_dungeon_rewards(ganon_bosskey_rewards)) or
-        (shuffle_ganon_bosskey == 'tokens' and (Gold_Skulltula_Token, ganon_bosskey_tokens))) or
-        (shuffle_ganon_bosskey != 'stones' and shuffle_ganon_bosskey != 'medallions' and
-         shuffle_ganon_bosskey != 'dungeons' and shuffle_ganon_bosskey != 'tokens')",
+        (shuffle_ganon_bosskey == 'tokens' and (Gold_Skulltula_Token, ganon_bosskey_tokens)) or
+        (shuffle_ganon_bosskey == 'triforce' and won_triforce_hunt()) or
```

Comment:
Why is this change here?

Reply:
if i recall, the triforce hunt line was added because the pots in ganons tower would never be reachable in a triforce hunt so "All locations reachable" would fail to generate.

Reply:
Right, that makes sense.